Dockerfile

```diff
@@ -1,36 +1,36 @@
 # ------------------------------------------------------------------------------
 # Builder Stage
 # ------------------------------------------------------------------------------
-FROM golang:1.19-bullseye AS build
+FROM golang:1.19-alpine AS build
+
+RUN apk add --no-cache make
 
 WORKDIR /build
-ADD . /build
 
+COPY go.mod go.sum ./
 RUN go mod download
-RUN make build
+
+COPY Makefile main.go ./
+COPY static/ static/
+COPY pkg/ pkg/
+COPY .git/ .
+RUN CGO_ENABLED=0 make build
 
 # ------------------------------------------------------------------------------
-# Release Stage
+# Fetch wait-for
 # ------------------------------------------------------------------------------
-FROM debian:bullseye-slim
-
-RUN \
-  apt-get update && \
-  apt-get install -y ca-certificates openssl netcat curl gnupg lsb-release && \
```

Comment:
Missing ca-certs and openssl packages might have an impact on pgweb connectivity capabilities, I did run into similar issues before, however i don't recall the details.

Also, netcat and curl as super useful when you need to drop into container and troubleshoot things quickly.

Reply:
Should pgweb connect to external servers while running?